pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -53,16 +54,18 @@ class ModificationsExportDialog extends React.Component {
 
     handleSelect(event, active_tab) {
         this.setState({ active_tab });
+        clearTimeout(this.tId);
```

Comment:
This should also set `copied: false`, as that shows / toggles the green icon. The `clearTimeout` isn't actually needed but it just feels correct to do it. (Also it could introduce a race when a user clicks the button quickly after switching a tab).